src/path.jl

```diff
@@ -249,14 +249,14 @@ Base.isempty(path::AbstractPath) = isempty(parts(path))
 
```

Comment:
All uses of `string` here should become `String`.
Since we are not wanting to work with a readable representation of the object (which si what `string` should do)
but with a `String` version of the object, which is what `String` should do.
(at least in analogy to the `String(::AbstractString)` method docs)
Really it is more of a convert but that is too verbose, no?